OpenGL/Constructs/VAO.cs

```diff
@@ -589,7 +609,8 @@ private void DrawOGL2()
         {
             if (VertexCount == 0) return;
             BindAttributes(Program);
-            Gl.DrawElements(DrawMode, VertexCount, elementType, IntPtr.Zero);
+            IntPtr offset = (IntPtr)(Offset * GetElementSizeInBytes());
+            Gl.DrawElements(DrawMode, VertexCount, elementType, offset);
```

Comment:
Similar to the comment above, it doesn't have to be for this commit, but I think it might be worth caching the computation of offset (especially since I imagine offset is usually zero) since this happens in the Draw code.